post-processor/vsphere/post-processor.go

```diff
@@ -144,6 +144,17 @@ func (p *PostProcessor) generateURI() (*url.URL, error) {
 	return u, nil
 }
 
+func getEncodedPassword(u *url.URL) (string, bool) {
+	// filter password from all logging
+	password, passwordSet := u.User.Password()
+	if passwordSet && password != "" {
+		encodedUserPassword := strings.Split(u.User.String(), ":")
+		encodedPassword := encodedUserPassword[len(encodedUserPassword)-1]
```

Comment:
I think this is the same as:
```suggestion
		encodedPassword := strings.Split(u.User.String(), ":")[1]
```

Reply:
Yup: https://play.golang.org/p/xb322f8cKdl